docs/src/constrained/irregular_surrogates.md

```diff
@@ -0,0 +1,25 @@
+# Irregular Timeseries Surrogates
+
+
+To derive a surrogate for irregular time series we can use surrogate methods which are irrespective of the time axis like [`RandomShuffle`](@ref) or [`BlockShuffle`](@ref)
+or we need to use algorithms, which take the irregularity of the time axis into account.
+
+## LombScargle based surrogate
+
+The LS surrogate is a form of a constrained surrogate which takes the LombScargle periodogram to derive surrogates with similar phase distribution.
```

Comment:
This sentence is incomplete. _"to derive surrogates with similar phase distribution_". ... similar to what? I'm guessing you mean similar phase distribution as the original time series!